tests/performance_tests/README.md

```diff
@@ -486,7 +486,7 @@ Performance Test Basic Single Test:
 * `actions_data`                The json actions data file or json actions data description string to use
 * `actions_auths`               The json actions auth file or json actions auths description string to use, containting authAcctName to activePrivateKey pairs.
 * `peer_endpoint`               set the peer endpoint to send transactions to, default="127.0.0.1"
-* `port`                        set the peer endpoint port to send transactions to, default=9876
+* `ports`                       Comma separated list of peer endpoint ports to send transactions to, default=9876
```

Comment:
Shouldn't `peer_endpoint` and `ports` be replaced with:

 `  connection_pair_list  Comma separated list of endpoint:port combinations to send transactions to` now?